apps/crn-frontend/src/network/teams/Outputs.tsx

```diff
@@ -50,30 +60,62 @@ const OutputsList: React.FC<OutputsListProps> = ({
     filters,
     currentPage,
     pageSize,
-    teamId,
+    ...(draftOutputs
+      ? {
+          userAssociationMember,
+          associationId: teamId,
+          draftsOnly: true,
+        }
+      : {
+          teamId,
+        }),
   });
+
   const { numberOfPages, renderPageHref } = usePagination(
     result.total,
     pageSize,
   );
   const { client } = useAlgolia();
+  const authorization = useRecoilValue(authorizationState);
   const exportResults = () =>
-    algoliaResultsToStream<typeof RESEARCH_OUTPUT_ENTITY_TYPE>(
-      createCsvFileStream(
-        `SharedOutputs_Team${utils
-          .titleCase(displayName)
-          .replace(/[\W_]+/g, '')}_${format(new Date(), 'MMddyy')}.csv`,
-        { header: true },
-      ),
-      (paginationParams) =>
-        getResearchOutputs(client, {
-          filters,
-          searchQuery,
-          teamId,
-          ...paginationParams,
-        }),
-      researchOutputToCSV,
-    );
+    draftOutputs
+      ? squidexResultsToStream<ResearchOutputResponse>(
+          createCsvFileStream(
+            `SharedOutputs_Drafts_Team_${utils
+              .titleCase(displayName)
+              .replace(/[\W_]+/g, '')}_${format(new Date(), 'MMddyy')}.csv`,
+            { header: true },
+          ),
+          (paginationParams) =>
+            getDraftResearchOutputs(
+              {
+                filters,
+                searchQuery,
+                userAssociationMember,
+                associationId: teamId,
+                draftsOnly: true,
```

Comment:
Not a big deal, but I was wondering if we could remove this `draftsOnly` from here because it seems kind of weird having to pass `draftsOnly: true` to a function called `getDraftResearchOutputs`. 

Maybe we could do something like
```
export const useResearchOutputs = (
  options: ResearchOutputListOptions,
  published = true,
) => {
  (...)
throw (
      !published && 'userAssociationMember' in options
        ? getDraftResearchOutputs(options, authorization)
        : getResearchOutputs(client, options).then(
            (data): ListResearchOutputResponse => ({
              total: data.nbHits,
              items: data.hits,
            }),
          )
(...)
```

in `apps/crn-frontend/src/shared-research/state.ts` so we would not pass `draftsOnly` inside `option`

Reply:
I don't think it matters too much. The types make it explicit that draftsOnly can only be true for that method. Its useful for type discrimination